frontend/docker/Dockerfile.multistage

```diff
@@ -1,12 +1,10 @@
 ## Stage 1 : build with maven builder image
 FROM registry.access.redhat.com/ubi8/nodejs-14 AS build
-RUN npm install --global yarn
-COPY package.json yarn.lock ./
-RUN yarn --frozen-lockfile
+COPY package.json package-lock.json ./
+RUN npm ci
 COPY public ./public
-COPY tsconfig.json ./
 COPY src ./src
-RUN yarn build --frozen-lockfile
+RUN npm run build
```

Comment:
just curious - should this not be using npm ci to have it use dependencies found in lockfile rather than just taking latest/greatest dependency resolution ?

Reply:
I am using `npm ci`

Reply:
https://github.com/ia3andy/code.quarkus.io/blob/use-bit/frontend/docker/Dockerfile.multistage#L4